Userland/Games/Breakout/Game.cpp

```diff
@@ -171,9 +173,11 @@ void Game::keydown_event(GUI::KeyEvent& event)
     case Key_Escape:
         GUI::Application::the()->quit();
         break;
+    case Key_A:
```

Comment:
We should probably tag these with `[[fallthrough]]` to document the intention.
See: https://en.cppreference.com/w/cpp/language/attributes/fallthrough